src/Symfony/Component/Mailer/Bridge/Mailchimp/RemoteEvent/MailchimpPayloadConverter.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Mailer\Bridge\Mailchimp\RemoteEvent;
+
+use Symfony\Component\RemoteEvent\Event\Mailer\AbstractMailerEvent;
+use Symfony\Component\RemoteEvent\Event\Mailer\MailerDeliveryEvent;
+use Symfony\Component\RemoteEvent\Event\Mailer\MailerEngagementEvent;
+use Symfony\Component\RemoteEvent\Exception\ParseException;
+use Symfony\Component\RemoteEvent\PayloadConverterInterface;
+
+final class MailchimpPayloadConverter implements PayloadConverterInterface
+{
+    public function convert(array $payload): AbstractMailerEvent
+    {
+        if (!\in_array($payload['event'], ['send', 'deferral', 'hard_bounce', 'soft_bounce', 'delivered', 'open', 'click', 'spam', 'unsub', 'reject'], true)) {
+            throw new ParseException(sprintf('Unsupported event "%s".', $payload['event']));
+        }
+        if (\in_array($payload['event'], ['send', 'deferral', 'soft_bounce', 'hard_bounce', 'delivered', 'reject'], true)) {
+            $name = match ($payload['event']) {
+                'send' => MailerDeliveryEvent::RECEIVED,
+                'deferral', 'soft_bounce' => MailerDeliveryEvent::DEFERRED,
+                'hard_bounce' => MailerDeliveryEvent::BOUNCE,
```

Comment:
In BrevoPayloadConverter, soft_bounce and hard_bounce match `MailerDeliveryEvent::BOUNCE`, should we do same here ?

Reply:
I would say yes. A soft bounce is not deffered